### ISSUE TITLE
Trying to change any bit(1) column throws Error 500

### ISSUE THREAD

**Issue by tsouloftas** [phpmyadmin/phpmyadmin#16111]
### Describe the bug

While trying to change any bit(1) column within the Structure tab on any table, it throws a 500 error. Apache error log thankfully gathers the issue with the following:

```
PHP Fatal error:  Uncaught TypeError: htmlspecialchars_decode() expects parameter 1 to be string, null given in 
libraries/classes/Util.php:2863\nStack trace:\n#0 
libraries/classes/Util.php(2863): htmlspecialchars_decode()\n#1 
libraries/tbl_columns_definition_form.inc.php(339): PhpMyAdmin\\Util::convertBitDefaultValue()\n#2 
libraries/classes/Controllers/Table/StructureController.php(573): include('
...')\n#3 
libraries/classes/Controllers/Table/StructureController.php(1635): PhpMyAdmin\\Controllers\\Table\\StructureController->displayHtmlForColumnChange()\n#4 
libraries/classes/Controllers/Table/StructureController.php(257): PhpMyAdmin\\Controllers\\Table\\StructureController->getDataForSubmitMult()\n#5 
tbl_structure.php(68): PhpMyAdmin\\Controllers\\Table\\StructureController->indexAction()\n#6 {main}\n  thrown in 
libraries/classes/Util.php on line 2863
```

### To Reproduce

1. Go to 'Structure' within a table
2. Select any column with bit(1) datatype
3. Click on change

Please note that the table is InnoDB and has Collation latin1_swedish_ci, if that may play any role.

<img width="1441" alt="Screenshot 2020-05-06 at 22 50 38" src="https://user-images.githubusercontent.com/715899/81222539-19bc1880-8fed-11ea-85a3-e5509ff51b57.png">

Also, please note that the 500 error modal box (above) is shown only when the row is pre-selected and the change button beneath is clicked. If you try by clicking change on the actual same row, you are re-directed to the edit page but without any results.

<img width="775" alt="Screenshot 2020-05-06 at 23 06 19" src="https://user-images.githubusercontent.com/715899/81223156-3b69cf80-8fee-11ea-85ba-b417fe666397.png">

### Expected behavior

As usual, to be able to change any column details whatever its datatype.

### Screenshots

Screenshot Added

### Server configuration

- Operating system: Debian 10 (buster)
- Web server: Apache 2.4.38
- Database version: mariadb 1.10
- PHP version: 7.4.4
- phpMyAdmin version: 5.0.2

### Client configuration

- Browser: Google Chrome 80
- Operating system: macOS Mojave 10.14

### Additionals

According to Apache error log, it seems that `htmlspecialchars_decode()` expects parameter to be string but null is given in `/usr/share/pma/libraries/classes/Util.php:2863`

**Comment by williamdes** [comment]
Hello @tsouloftas 
Fortunately this issue is a duplicate of #16048 that is now fixed (https://github.com/phpmyadmin/phpmyadmin/issues/16048#issuecomment-608269062)

> ae640ebcce4fffdc489f3abbdaf53353e4c75e8e is the fix and I added a test case aab8c19731b54d629a669ab97c3be01fb495915f  


Thank you for the report!
You can download the latest version in development [(phpMyAdmin 5.0+snapshot)](https://www.phpmyadmin.net/downloads/) from our website before 5.0.3 is available.

**Comment by tsouloftas** [comment]
Thank you for your early response. I am using the latest stable version 5.0.2.

You mean I should download the development version 5.0.3?

**Comment by tsouloftas** [comment]
Very well thank you! Will the fix be available in 5.0.3 stable?

**Comment by williamdes** [comment]
> Very well thank you! Will the fix be available in 5.0.3 stable?

:)
Yes it will be
